source/cpuid/cpuid.go

```diff
@@ -13,26 +13,12 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+// +build amd64
```

Comment:
Remove this. It has no effect in this location (should be at the top of the file), and, this file should be compiled for all archs